map/download-preplanned-map/src/main/java/com/esri/samples/download_preplanned_map/PreplannedMapAreaListCell.java

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2019 Esri
```

Comment:
one comment for this: was the name of this class supposed to be
`DownloadPreplannedMapAreaListCell` ?

Reply:
I think the name is correct as is `PreplannedMapAreaListCell`, because it just represents a cell item which holds a preplanned map area. It doesn't have anything to do directly with downloading the areas.

Reply:
nice one, just wanted to check thanks!